src/styles/_global-nav.scss

```diff
@@ -462,6 +462,13 @@ a.nav-skip-button {
   display: none;
 }
 
+// Ensures header does not expand after opening the dropdown menu and expanding the window view size
+@media (min-width: $screen-sm) {
+  .global-nav--component .global-nav--row-desktop {
+    height: 68.5px;
```

Comment:
Where is that number coming from? It seems rather random 😄?

Reply:
It might be a good idea to set a variable for the height and use it across different screen widths. The use case seems to be fixed for me now though.

Reply:
@rsylvian The height is from the height of the `global-nav--component` when the window size is > 768px. Basically just making sure the header stays at that height when the window expands back to > 768px.

Reply:
@chantelle11 unfortunately that another bug that we talked about :( I made a ticket for it in our backlog [here](https://circleci.atlassian.net/browse/DD-355)

Reply:
Sounds good!